lib/jelix-modules/jacl2db/plugins/acl2/db/db.acl2.php

```diff
@@ -94,6 +94,35 @@ public function getRight($subject, $resource='-'){
             return true;
     }
 
+    /**
+     * return the value of the manage groupright on the given subject (and on the optional resource)
+     * @param string $subject the key of the subject
+     * @param string $resource the id of a resource
+     * @return boolean true if the right is ok
+     */
+    public function getManageGroupRight($subject, $resource = '-')
+    {
+        if (empty($resource))
+            $resource = '-';
+
+        if (!jAuth::isConnected()) {
+            return self::getAnonymousRight($subject, $resource);
+        }
+
+        // let's load all rights for the groups on which the current user is attached
+        $groups = jAcl2DbUserGroup::getGroups();
+
+        if (count($groups)) {
+            $dao = jDao::get('jacl2db~jacl2rights', 'jacl2_profile');
+            $right = $dao->getRightWithRes($subject, $groups, $resource);
+
+            if ($right === null) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
I don't really understand what this method returns. It returns true if there are NOT groups, but also returns true when there groups with the given subject/resource. It does not make sens IMHO.